### PR TITLE
2022q1 Intel wireless driver and LinuxKPI 802.11 compatibility as well as Realtek wireless drivers reports

### PR DESCRIPTION
**Pull request by bzfbd**

### CODE REVIEW

File: 2022q1/iwlwifi.adoc [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+Some of the changes made while porting the driver to FreeBSD were kindly incorporated into the upstream Linux driver already.
+
+During the first quarter work continued with about 70 commits.
+Updating the driver and firmware reduced differences to the Linux version and gave us bugfixes and improvements.
+Changes to the LinuxKPI 802.11 compatibility layer were made to avoid firmware crashes and possible panics for users along with other improvements.
+
+Auto-loading support for LinuxKPI PCI drivers was comitted.
+This means that iwlwifi(4) will now load automatically during boot if a supported card is detected without any user interactions.
+Considering the current state of the driver and the next release a decision was made that iwm(4) supported chipsets will continue to attach to iwm(4) for now and only newer and otherwise unsupported chipsets will use the iwlwifi(4) driver.
+This is likely going to change in CURRENT as soon as iwlwifi(4) will provider better support than iwm(4).
```
Comment (pauamma) [inline 2022q1/iwlwifi.adoc:19]: s/will provider/provides/

**Review by pauamma — CHANGES_REQUESTED** [review]

File: 2022q1/rtw88.adoc [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+link:https://wiki.freebsd.org/WiFi/Rtw89[rtw89 status FreeBSD wiki page] URL: link:https://wiki.freebsd.org/WiFi/Rtw89[https://wiki.freebsd.org/WiFi/Rtw89]
+
+Contact: Bjoern A. Zeeb <bz@FreeBSD.ORG>
+
+While the Intel Wireless driver update project is the main driver behind the work to bring support for newer chipsets and eventually newer IEEE 802.11 standards support, there is also an ongoing effort to support more drivers.
+The next two drivers in the (already longer) queue are Realtek's rtw88 and rtw89.
+
+While the initial driver porting efforts for rtw88 and rtw89 happened on personal time, the LinuxKPI integration has to be done more and more along the Intel wireless driver work and so thanks are also due to the FreeBSD Foundation.
+
+The rtw88 driver has started to work on machines with less than 4GB of main memory and should (despite that problem) be in the FreeBSD git repository by the time this report is published.
```
Comment (pauamma) [inline 2022q1/rtw88.adoc:14]: Clarify the problem? Is it "doesn't work reliably with less than 4GB"? Or is it how/whether it work with more than that?

File: 2022q1/rtw88.adoc [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+While our version of the driver is aware of these limitations, the problem is currently assumed to be outside the driver in the interactions with LinuxKPI and busdma.
+
+The rtw89 driver has happily started to send packets and has troubles receiving at this point.
```
Comment (pauamma) [inline 2022q1/rtw88.adoc:17]: s/troubles/trouble/ (or "problems", if you prefer keeping the plural)

File: 2022q1/rtw88.adoc [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+More investigations will happen as soon as rtw88 is sorted out and it is expected that rtw89 will then also timely follow into FreeBSD's git repository.
```
Comment (pauamma) [inline 2022q1/rtw88.adoc:18]: I think you want the collective singular ("investigation") here unless you already know there are unrelated things that will need separate investigations, one for each.

**Review by pauamma — CHANGES_REQUESTED** [review]

**Comment by bzfbd** [comment]
I am lost as to how to see the right version and mark your requested changes as done.   Thanks a lot for the review.  I hope I didn't add more when updating the text.

**Review by pauamma — APPROVED** [review]

**Pull request merged**